Drupal/README.md

```diff
@@ -0,0 +1,137 @@
+# Composer template for Drupal projects
```

Comment:
We talked briefly about replacing this README with one that's specific to our project - here are my ideas for things that are important to include (based on my experience pulling down your changes and getting the app up and running):

- Pre-requisite that `composer` is installed (from https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos).
- The steps to get the app up and running:
   * `cd` into the `Drupal` directory
   * `lando start`
   * `lando composer install`
   * `lando db-import database_export_file.gz`
   * (optional) `lando info` (to get the localhost port the app is running on)
- A reminder that we should always do `lando db-export database_export_file` when creating a PR to make changes to the app (until we've figured out how to move the config from the DB to code)
- I think it might also be helpful to just include a link to the guide you followed when creating this app (for reference)